composer.json

```diff
@@ -11,13 +11,15 @@
     },
     "require": {
         "php": ">=5.6",
-        "roave/security-advisories": "dev-master",
-        "silinternational/email-service-php-client": "^2.0.0",
```

Comment:
Moving this from a requirement to a suggestion looks good to me, though I think it will merit a major version bump (since anyone using that adapter will now need to separately specify that as a requirement, thus making this a potentially breaking change). @fillup, any objection to me proceeding along those lines?